lib/liftoff/cli.rb

```diff
@@ -18,7 +18,8 @@ def parse_command_line_options
 
     def global_options
       OptionParser.new do |opts|
-        opts.banner = 'usage: liftoff [-v | --version] [-h | --help] [config options]'
+        opts.banner = 'usage: liftoff [-v | --version] [-h | --help] ' \
```

Comment:
Don't worry about breaking this line up just to appease Hound.

![](http://dl.dropboxusercontent.com/u/452364/gifs/fuckthepolice.gif)